### PR TITLE
Azure Analysis Services: added support for new SKUs

### PR DESCRIPTION
**Pull request by igor-uzhviev**
## Description
Added new SKUs for Azure Analysis Services: B1, B2, S0.
Pull requests for API spec changes: https://github.com/Azure/azure-rest-api-specs/pull/1109
and https://github.com/Azure/azure-rest-api-specs/pull/1184

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by shahabhijeet** [comment]
@azuresdkci add to whitelist

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

**Review by shahabhijeet — CHANGES_REQUESTED** [review]
I don't see any changes to the version.
Please update the version of the nuget inside .csproj as well as version info in AssemblyInfo.cs
If there are any legacy project.json, global.json and *.xproj, delete them for your project.

**Comment by shahabhijeet** [comment]
@azuresdkci test this please

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs [hunk 1]
```diff
@@ -14,6 +14,9 @@ namespace Microsoft.Azure.Management.Analysis.Models
     /// </summary>
     public static class SkuName
     {
+        public const string B1 = "B1";
+        public const string B2 = "B2";
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs]: @igor-uzhviev you will need to bump up the version in .csproj and AssemblyInfo.cs

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

**Comment by markcowl** [comment]
@azuresdkci add to whitelist

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs (hunk 1 above)
Comment (igor-uzhviev) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs]: done.

**Comment by shahabhijeet** [comment]
@igor-uzhviev do you already have existing tests for SKU's?

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs (hunk 1 above)
Comment (markcowl) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs]: Can we add one of the new skus to an existing test?

**Review by markcowl — CHANGES_REQUESTED** [review]

File: src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs (hunk 1 above)
Comment (igor-uzhviev) [inline src/SDKs/AnalysisServices/Management.Analysis/Generated/Models/SkuName.cs]: I would really prefer not to do it now. There is just one scenario test which tests multiple actions one by one (provision->update->delete), it does not really test different skus, it just provisions a server with first sku. We have already many sku constants defined and I just added extra more. In the near future we want to switch to dynamic skus loading when they will be loaded from REST call, then we will definitely implement more advanced sku testing.

**Comment by igor-uzhviev** [comment]
@markcowl a new test was added. Please check.

**Comment by igor-uzhviev** [comment]
Travis CI build failed. The error is:
`/home/travis/.travis/job_stages: line 53: ./testSdkOnLinux.sh: Permission denied`
Seems to be infrastructure related...

**Comment by shahabhijeet** [comment]
@igor-uzhviev so ignore travis build, that is still getting worked on.
But Jenkins failed and AnalysisService tests failed.

File: src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj [hunk 2]
```diff
@@ -26,6 +26,9 @@
     <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\CreateGetUpdateDeleteTest.json">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Update="SessionRecords\AnalysisServices.Tests.ScenarioTests.ServerOperationsTests\ScaleUpTest.json">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```
Comment (shahabhijeet) [inline src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj]: @igor-uzhviev please remove commented code from all csproj files.

**Review by shahabhijeet — CHANGES_REQUESTED** [review]

File: src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj (hunk 2 above)
Comment (igor-uzhviev) [inline src/SDKs/AnalysisServices/AnalysisServices.Tests/AnalysisServices.Tests.csproj]: @shahabhijeet done.

**Review by shahabhijeet — APPROVED** [review]

**Pull request merged**